core/src/commonMain/kotlin/com/revenuecat/purchases/kmp/models/GoogleReplacementMode.kt

```diff
@@ -62,13 +62,3 @@ public expect enum class GoogleReplacementMode: ReplacementMode {
      */
     DEFERRED;
 }
-
-public val GoogleReplacementMode.playBillingClientMode: Int
```

Comment:
Same, not really needed by consumers since we're not using PHC to map these values and we're using `purchases-android`'s purchase methods directly.